packages/flutter_localizations/lib/src/l10n/cupertino_pt.arb

```diff
@@ -0,0 +1,22 @@
+{
+  "todayLabel": "Hoje",
+  "datePickerHourSemanticsLabelOne": "$hour em ponto",
```

Comment:
"$hour hora"
```suggestion
  "datePickerHourSemanticsLabelOne": "$hour hora",
```

Reply:
Thanks for diff'ing and reviewing. We'll backport these into the translation console

Reply:
I assume these are both for brazilian and portugal portuguese?

Reply:
Yes, the use of "em ponto" is not for this situation.

The use wold be for something like 8 o'clock (8 "horas em ponto").